README.md

```diff
@@ -249,6 +249,7 @@ Table of Contents
    * https://parsehub.com/ — Extract data from dynamic sites, turn dynamic websites into APIs, 5 projects free.
    * https://import.io/ - Easily turn websites into APIs, completely free for life.
    * https://redsmin.com/ - Online real-time monitoring and administration service for Redis, 1 Redis instance free
+   * https://cloud.google.com/datastore/ - High scale NoSQL datastore, free daily quota: 1GB storage + 50K reads + 50K writes
```

Comment:
this is already listed under PaaS as part of the larger google cloud platform entry